test/test.hs

```diff
@@ -2448,6 +2417,82 @@ instance Arbitrary (LitOnly (Expr EWord)) where
 instance Arbitrary (LitOnly (Expr Buf)) where
   arbitrary = LitOnly . ConcreteBuf <$> arbitrary
 
+-- GenWordSimplifyExpr
+newtype GenWordSimplifyExpr a = GenWordSimplifyExpr a
+  deriving (Show, Eq)
+
+instance Arbitrary (GenWordSimplifyExpr (Expr EWord)) where
+  arbitrary = do
+    fmap GenWordSimplifyExpr . sized $ genWord 0
+
+-- GenWriteWordBuf
+newtype GenWriteWordBuf a = GenWriteWordBuf a
+  deriving (Show, Eq)
+
+instance Arbitrary (GenWriteWordBuf (Expr Buf)) where
```

Comment:
I wonder if we should just make this the default arbitrary instance, and then have a special one (`SparseWritesBuf`) or something if we need the existing default `Arbitrary` instance for `Expr Buf`?

Reply:
We decided it's not so important so I'm resolving this.